### ISSUE TITLE
Build error in ios : Did not find header 'renderer/animations/primitives.h' in framework 'react' 

### ISSUE THREAD

**Issue by nateshmbhat** [payu-intrepos/payu-non-seamless-react#66]
Getting the following build error when building on ios : 

I'm using 3.0.11 version 

Showing All Errors Only
/Users/natesh.bhat/Desktop/isha/sgapp-copy/node_modules/react-native/ReactCommon/react/renderer/animations/utils.h:10:10: Did not find header 'renderer/animations/primitives.h' in framework 'react' (loaded from '/Users/natesh.bhat/Library/Developer/Xcode/DerivedData/Sadhguru-cerqxzitwdajbbgdtujoqflyemwv/Build/Products/Debug-iphonesimulator/React-Core')

<img width="989" alt="Screenshot 2024-05-07 at 5 19 03 PM" src="https://github.com/payu-intrepos/payu-non-seamless-react/assets/23279926/8f964f81-ce21-4491-8862-f2e062f79b04">

**Comment by kjp151087** [comment]
I am also getting same issue now. I have updated my react native version to `0.74.1` and payu-non-seam-less-react version `3.0.10`.

**Comment by nateshmbhat** [comment]
i also started getting this error when i updated to react native version 0.73 and then 0.74.1

**Comment by nateshmbhat** [comment]
@shahfaizalc can you please check this issue ?

**Comment by evil25** [comment]
Did you get solution @nateshmbhat for this i am also facing same issue, i am using 0.73 version and library version is 3.0.12

**Comment by nateshmbhat** [comment]
no solution yet @evil25

**Comment by kkkasio** [comment]
I have same problem, I update react native to "0.73.8" (from 0.66.5 ><)

<img width="1140" alt="Captura de Tela 2024-06-30 às 23 10 09" src="https://github.com/payu-intrepos/payu-non-seamless-react/assets/19254402/8e8dee67-379d-4ae7-a6c1-63ff7e49d841">

**Comment by keyline** [comment]
Update utils.h file :

 #ifdef __cplusplus
 #pragma once
.....
.........
....
....
} 
#endif

**Comment by RajuPedimalla** [comment]
Where Can I find the utils.h file and What is the code I need to update

**Comment by akansha1907** [comment]
facing same issue
My react-native version and payu version are
  "payu-non-seam-less-react": "^3.1.3-alpha1"
    "react-native": "0.73.4"
    
<img width="723" alt="Screenshot 1946-06-19 at 10 06 15 AM" src="https://github.com/user-attachments/assets/c65c1bf8-ba4b-4aec-ba79-d174779e476e">

**Comment by akansha1907** [comment]
> Update utils.h file :
> 
> #ifdef __cplusplus #pragma once ..... ......... .... .... } #endif

**Comment by akansha1907** [comment]
I changed payu-non-seam-less-react version from "^3.1.3-alpha1" to  "payu-non-seam-less-react": "^3.0.7" By doing this my error was resolved, and I was able to make IOS build successfully

**Comment by RajuPedimalla** [comment]
Yes, thanks akansha.... "payu-non-seam-less-react":^3.0.7 is working fine.... IOS build successful and payment is done successfully in the real device..
